unittests/SubscriptionManagerTests.cpp

```diff
@@ -190,19 +192,16 @@ TEST_F(SubscriptionManagerSimpleBackendTest, ReportCurrentSubscriber)
 
 TEST_F(SubscriptionManagerSimpleBackendTest, SubscriptionManagerLedgerUnSub)
 {
-    MockBackend* rawBackendPtr = dynamic_cast<MockBackend*>(mockBackendPtr.get());
-    ASSERT_NE(rawBackendPtr, nullptr);
-    mockBackendPtr->updateRange(10);  // min
-    mockBackendPtr->updateRange(30);  // max
+    backend->setRange(10, 30);
     boost::asio::io_context ctx;
     auto ledgerinfo = CreateLedgerInfo(LEDGERHASH, 30);
     // mock fetchLedgerBySequence return this ledger
-    ON_CALL(*rawBackendPtr, fetchLedgerBySequence).WillByDefault(Return(ledgerinfo));
-    EXPECT_CALL(*rawBackendPtr, fetchLedgerBySequence).Times(1);
+    ON_CALL(*backend, fetchLedgerBySequence).WillByDefault(Return(ledgerinfo));
+    EXPECT_CALL(*backend, fetchLedgerBySequence).Times(1);
```

Comment:
Maybe if you are already changing this you could also change `ON_CALL(); EXPECT_CALL().Times(1)` to `EXPECT_CALL()`?
Not important but would look nicer if you can be bothered 🙂

Reply:
This is more than 500 places to change by the look of things.. a quick regex did not help so i'm gonna leave this for another time 👍